### PR TITLE
DB setup

### PR DESCRIPTION
**Pull request by TempusShift**

### CODE REVIEW

**Review by JakeDarling — APPROVED** [review]

**Review by kmb4577 — APPROVED** [review]

File: knockserver/models.py [hunk 1]
```diff
@@ -3,7 +3,31 @@
 from knockserver.database import Base
 
 class ExampleObject(Base):
```
Comment (corbanmailloux) [inline knockserver/models.py]: @ajd4223 Could you explain what the `ExampleObject` is for? Or is it just from the original sample project?

Comment (TempusShift) [reply knockserver/models.py]: It is from the base
I just didnt remove it.

Comment (TempusShift) [reply knockserver/models.py]: For if we for some reason wish to use it as a reference again.

Comment (corbanmailloux) [reply knockserver/models.py]: Can we at least comment it out? I don't really want `ExampleObject` in our DB.

**Pull request merged**